components/logo/MainLogo.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useUser, useSubscription, useOrganization, Href } from 'react-components';
+
+import { formatPlans } from '../../containers/payments/subscription/helpers';
+
+const MainLogo = ({ product, url = 'https://mail.protonmail.com/' }) => {
+    const [{ isFree, hasPaidMail, hasPaidVpn } = {}, loadingUser] = useUser();
+    const [{ Theme } = {}, loadingOrganization] = useOrganization();
+    const [{ Plans } = {}, loadingSubcription] = useSubscription();
+
+    const { mailPlan = {}, vpnPlan = {} } = formatPlans(Plans);
```

Comment:
Don't use `formatPlans`, create a helper in `proton-shared` or extend `useSubscription` hook as we made on `useUser` to know `hasPlus`, `hasVisionary`, `hasProfessional`, etc ...

Reply:
After talking to @mmso , we decided it was simpler to keep `formatPlans`